component/ci-base/Dockerfile

```diff
@@ -33,6 +33,12 @@ RUN set -eux; \
         docker-buildx-plugin \
         docker-compose-plugin \
         xvfb \
+        libgtk2.0-0 \
+        libnss3 \
+        libatk-bridge2.0-0 \
+        libgtk-3-0 \
+        libgdm-dev \
+        libasound2 \
```

Comment:
At some point, we should group these into a `CYPRESS_DEPS` variable or something similar so that we know what `cypress` specifically needs.